### PR TITLE
Remove ChromeProxyService.setExceptionPauseMode()

### PR DESCRIPTION
**Pull request by derekxu16**

### CODE REVIEW

File: dwds/lib/src/services/chrome_proxy_service.dart [hunk 1]
```diff
@@ -776,15 +776,10 @@ ${globalLoadStrategy.loadModuleSnippet}("dart_sdk").developer.invokeExtension(
...
-  }
-
-  @override
-  Future<Success> setExceptionPauseMode(String isolateId, String mode) async {
     await isInitialized;
-    _checkIsolate('setExceptionPauseMode', isolateId);
-    return (await debuggerFuture).setExceptionPauseMode(mode);
+    _checkIsolate('setIsolatePauseMode', isolateId);
+    return (await debuggerFuture)
+        .setExceptionPauseMode(exceptionPauseMode ?? ExceptionPauseMode.kNone);
```
Comment (derekxu16) [inline dwds/lib/src/services/chrome_proxy_service.dart:782]: Do you think `Debugger.setExceptionPauseMode()` should be renamed as well?

File: dwds/test/chrome_proxy_service_test.dart [hunk 2]
```diff
@@ -1135,25 +1135,6 @@ void main() {
         expect(stack.truncated, isFalse);
       });
 
-      test('break on exceptions with legacy setExceptionPauseMode', () async {
```
Comment (elliette) [inline dwds/test/chrome_proxy_service_test.dart:1138]: Can we keep this test and instead have it test "break on exceptions with `setIsolatePauseMode?`"

File: dwds/lib/src/services/chrome_proxy_service.dart (hunk 1 above)
Comment (elliette) [reply dwds/lib/src/services/chrome_proxy_service.dart:782]: No, I don't think it needs to be

File: dwds/test/chrome_proxy_service_test.dart (hunk 2 above)
Comment (derekxu16) [reply dwds/test/chrome_proxy_service_test.dart:1107]: You already have a test below this one for "break on exceptions with setIsolatePauseMode". I just double-checked for any differences in it though, and found a missing line, so I just added it.

File: dwds/test/chrome_proxy_service_test.dart [hunk 3]
```diff
@@ -1492,17 +1473,25 @@ void main() {
       await expectLater(service.reloadSources(''), throwsRPCError);
     });
 
-    test('setExceptionPauseMode', () async {
+    test('setIsolatePauseMode', () async {
       final vm = await service.getVM();
       final isolateId = vm.isolates!.first.id!;
-      expect(await service.setExceptionPauseMode(isolateId, 'all'), _isSuccess);
-      expect(await service.setExceptionPauseMode(isolateId, 'unhandled'),
+      expect(
```
[the remark this answers is not in the document]
Comment (derekxu16) [reply dwds/test/chrome_proxy_service_test.dart:1451]: Sorry, can you please elaborate?

[the remark this answers is not in the document]
Comment (elliette) [reply dwds/test/chrome_proxy_service_test.dart:1451]: I think the request is to add a test case that calling `setIsolatePauseMode` without any of the optional parameters succeeds:

`await service.setIsolatePauseMode(isolateId), _isSuccess)`

**Review by elliette — APPROVED** [review]
LGTM once the extra test case is added!

**Pull request merged**